gtsfm/loader/colmap_loader.py

```diff
@@ -112,8 +99,8 @@ def __len__(self) -> int:
         """
         return self._num_imgs
 
-    def get_image(self, index: int) -> Image:
-        """Get the image at the given index.
+    def get_image_full_res(self, index: int) -> Image:
+        """Get the image at the given index, at full resolution.
```

Comment:
I'm not sure we need two different functions for the same thing? You could just have `max_resolution: Optional[int] = 760` in the base loader, and if it's `None`, then there is no max resolution and the images don't need to be resized.

Reply:
I like this suggestion. I think it's another valid way of doing it. I think we want to upper bound it though, since I think 4k is just not possible. e.g. so we should never allow above 4k. So I think I prefer it more as it is.